chainntnfs/txconfnotifier.go

```diff
@@ -177,91 +188,106 @@ func (tcn *TxConfNotifier) UpdateConfDetails(txid chainhash.Hash,
 
 	// First, we'll determine whether we have an active notification for
 	// this transaction with the given ID.
-	ntfns, ok := tcn.confNotifications[txid]
+	confSet, ok := tcn.confNotifications[txid]
 	if !ok {
-		return fmt.Errorf("no notifications found for txid %v", txid)
+		return fmt.Errorf("no notification found with TxID %v", txid)
 	}
 
-	ntfn, ok := ntfns[clientID]
+	ntfn, ok := confSet.ntfns[clientID]
 	if !ok {
 		return fmt.Errorf("no notification found with ID %v", clientID)
 	}
 
+	// Decrement the rescan count for each registered client that finishes.
+	confSet.rescanCount--
```

Comment:
since this is removed in a later commit, they should be squashed together to not having to introduce it in the first place.

Reply:
fixed